fragments/labels/bracketsio.sh

```diff
@@ -0,0 +1,7 @@
+adobebrackets|bracketsio)
```

Comment:
can you change this to be in two lines:

```
adobebrackets|
bracketsio)
```

the other format breaks the algorithm to list the labels...

Reply:
Done, any suggestions on how to resolve the windows only update issue?